src/consensus/aft/test/driver.h

```diff
@@ -732,13 +756,15 @@ class RaftDriver
 
       if (should_send)
       {
-        log_msg_details(src, dst, contents);
         _nodes.at(dst).raft->recv_message(
           src, contents.data(), contents.size());
         return true;
       }
     }
 
+    // Log that this message was dropped
+    log_msg_details(src, dst, contents, true);
```

Comment:
There are _2_ places where we "drop" messages in the driver. One is for a `drop_pending_to` action, handled explicitly above. But we also allow connections to be severed, in which case we essentially drop it here (early-out in `send_single_message`, when the required connection doesn't exist). We log for both, so that (hopefully) every change to `messages` has a corresponding traceable event.